app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -19,7 +19,10 @@ def send_code
       result = otp_delivery_selection_form.submit(delivery_params)
       analytics.otp_delivery_selection(**result.to_h)
       if result.success?
-        handle_valid_otp_params(user_select_delivery_preference, user_selected_default_number)
+        handle_valid_otp_params(
+          result, user_select_delivery_preference,
+          user_selected_default_number
```

Comment:
Can we format one argument per line? (I've been thinking to enforce this in Rubocop, since I personally find it can be hard to identify the arguments when they're not consistently either all on the same line or one-per-line)

```suggestion
          result,
          user_select_delivery_preference,
          user_selected_default_number,
```

Reply:
yeah, I agree, would be great to have rubocop enforce it

Reply:
I have a branch sitting around somewhere, I'll dig it up.